app/filtering.js

```diff
@@ -364,6 +364,12 @@ function registerForHeadersReceived (session, partition) {
       muonCb({ cancel: true })
       return
     }
+
+    if (firstPartyUrl !== details.url) {
```

Comment:
please use `isThirdPartyHost` instead on the URL hostnames since we want to check this based on the host, not the full URL